src/tigerbeetle.zig

```diff
@@ -6,32 +6,34 @@ const stdx = @import("./stdx.zig");
 
 pub const Account = extern struct {
```

Comment:
Also need to update `docs/DESIGN.md` with these account/transfer schema changes.